src/UE4SSProgram.cpp

```diff
@@ -1074,6 +1079,15 @@ namespace RC
         setup_mods();
         start_lua_mods();
         start_cpp_mods();
+        
+        if (Unreal::UnrealInitializer::StaticStorage::bIsInitialized) {
```

Comment:
```suggestion
        if (Unreal::UnrealInitializer::StaticStorage::bIsInitialized)
        {
```
Code-style consistency.